packages/flutter/test/services/platform_channel_test.dart

```diff
@@ -170,13 +171,25 @@ void main() {
       expect(result, isNull);
     });
 
-    test('can handle method call with no registered plugin', () async {
+    test('can handle method call with no registered plugin (setting before)', () async {
       channel.setMethodCallHandler(null);
       final ByteData call = jsonMethod.encodeMethodCall(const MethodCall('sayHello', 'hello'));
       ByteData envelope;
       await ServicesBinding.instance.defaultBinaryMessenger.handlePlatformMessage('ch7', call, (ByteData result) {
         envelope = result;
       });
+      await null; // just in case there's something async happening
+      expect(envelope, isNull);
+    });
+
+    test('can handle method call with no registered plugin (setting after)', () async {
```

Comment:
FWIW the title for this test is confusing to me.  "registered plugin"?  "registered handler?"

Reply:
heh, this is one reason why in my ideal test framework there's no test names. :-)

I think it refers to what happens when a method call comes in but no plugin has registered a handler for that channel.

Reply:
(the name is identical to the previous test, all I did was make a copy of the test with a minor tweak)

Reply:
Feel free to change that one too, it's equally confusing.